src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -74,7 +74,10 @@ public function build(ContainerBuilder $container)
         parent::build($container);
 
         $container->addCompilerPass(new RoutingResolverPass());
-        $container->addCompilerPass(new ProfilerPass());
+        if (class_exists(AddConsoleCommandPass::class)) {
+            $container->addCompilerPass(new AddConsoleCommandPass());
+        }
+        $this->addCompilerPassIfExists($container, ProfilerPass::class);
```

Comment:
@xabbuh In fact this doesn't expect a fallback. For the AddConsoleCommandPass, we just added a conflict to the frameworkbundle so that it is always available. I started to do the same for the FormPass (#21283) and SerializerPass(#21293), is it ok to do the same here or should we fallback to the deprecated one and mute deprecation instead?